manifests/init.pp

```diff
@@ -427,6 +430,11 @@
   Class['pulp::install'] -> Class['pulp::config'] -> Class['pulp::database'] ~> Class['pulp::service', 'pulp::apache']
   Class['pulp::config'] ~> Class['pulp::service', 'pulp::apache']
 
+  if $yum_gpg_sign_repo_metadata {
+    include ::pulp::repomd_signatures
+    Class['pulp::repomd_signatures']
```

Comment:
Isn't `contain ::pulp::repomd_signatures` equivalent to this?

Reply:
Yes, looks like it is...  Will change.